lib/service/passkey_service.dart

```diff
@@ -128,10 +143,34 @@ class PasskeyServiceImpl implements PasskeyService {
       'credentialCreationResponse':
           _registerResponse!.toCredentialCreationResponseJson(),
     });
-    await _userAccountChannel.setDidRegisterPasskey(true);
-    await _userAccountChannel.setUserId(addressAuthentication['requester']);
+    await setDidRegisterPasskey(true);
     _authService.setAuthToken(response);
   }
+
+  @override
+  Future<bool> didRegisterPasskey() async {
+    if (Platform.isAndroid) {
+      return _configurationService.didRegisterPasskey();
```

Comment:
if the function `didRegisterPasskey` is specific for android, could you please add android in func name